CMakeLists.txt

```diff
@@ -138,4 +138,10 @@ endif()
 
 if (TESTS AND NOT EMSCRIPTEN)
 	add_subdirectory(test)
+
+	add_custom_target(lsptest
```

Comment:
What is the purpose of this?

Reply:
I think I was removing this. Did it accidentally resurrect? The purpose is/was that it's easier to type `make lsptest` then fully qualifying the path. History scrolling wouldn't be faster either.  Plus the benefit that it auto-rebuilds solc if needed.

Reply:
We don't have anything like that anywhere else, so I would prefer to remove it.